watertap/unit_models/electrodialysis_1D.py

```diff
@@ -658,7 +658,7 @@ def build(self):
             self.flowsheet().time,
             self.diluate.length_domain,
             initialize=0.9,
-            bounds=(0, 1),
+            bounds=(0, 1 + 1e-10),
             units=pyunits.dimensionless,
             doc="The overall current efficiency for deionizaiton",
```

Comment:
```suggestion
            doc="The overall current efficiency for deionization",
```
totally unrelated to your changes